### PR TITLE
Updated CollectBenchmarkSucceeded utility

### PR DESCRIPTION
**Pull request by rickymagner**
Updated CollectBenchmarkSucceeded utility to match new Terra workspace bucket path paradigms for where workflow outputs are stored. Also added brief documentation on the `IndexCramOrBam` WDL.

### CODE REVIEW

**Review by michaelgatzen — COMMENTED** [review]
Should we maybe use the firecloud API instead for future-proofing?

**Comment by rickymagner** [comment]
Added Firecloud API code to help "future-proof" against changing Terra conventions. Sorry this has grown past a 30 second review, but appreciate you taking a look!

**Review by michaelgatzen — APPROVED** [review]
Looks good!

**Pull request merged**
